jenkins/release.JenkinsFile

```diff
@@ -9,7 +9,7 @@ standardReleasePipelineWithGenericTrigger(
     downloadReleaseAsset: true,
     publishRelease: true) {
         publishToRubyGems(
-            publicCertPath: "${WORKSPACE}/.github/opensearch-rubygems.pem",
+            publicCertPath: ".github/opensearch-rubygems.pem",
```

Comment:
My bad! Should have checked https://github.com/opensearch-project/opensearch-build-libraries/blob/main/vars/publishToRubyGems.groovy#L22